setup.py

```diff
@@ -154,6 +154,10 @@ def find_platform_sdk_dir():
         key = winreg.OpenKey(winreg.HKEY_LOCAL_MACHINE,
                               r"SOFTWARE\Microsoft\Windows Kits\Installed Roots")
         installRoot = winreg.QueryValueEx(key, "KitsRoot81")[0]
+    except EnvironmentError:
```

Comment:
Thanks, and sorry for the delay, but the right way to check that node is indeed different - have a look at `WinExt_win32com_mapi()` in the same file, but in short, you would pass 4 params to `winreg.OpenKey()`, where the 4th param is `winreg.KEY_READ | winreg.KEY_WOW64_32KEY`, with an additional complication that `winreg.KEY_WOW64_32KEY` doesn't exist on 32bit windows.